### PR TITLE
Update documentation for getting setup with local development

### PR DESCRIPTION
**Pull request by bfreeds**
This PR:

- Updates the documentation for `LOCALDEV.md` and `DOCKER.MD`
- Adds `db:schema:load` and `db:seed` to the `docker-compose` setup
- Ensures everything points to the same ruby version

### CODE REVIEW

File: LOCALDEV.md [hunk 1]
```diff
@@ -52,16 +52,18 @@ CEQR App has two Postgis databases:
 Rails config sits in a `.env` file that is not checked into version control. This allows configuring through environment variables in production. To get started, copy the config example:
 
 ```
+cd backend
 cp .env-example .env
 ```
 
 Next, edit `.env` to the appropriate settings. Descriptions of each are below:
 
+- `DATABASE_URL` - Postgres url string to the `ceqr_rails` database. Locally, this should point to your develompent database. Rails will automatically swap to `ceqr_test` when the test suite is run. Remember to use `postgis://` as the prefix.
```
Comment (godfreyyeung) [inline LOCALDEV.md]: nit: development

File: LOCALDEV.md [hunk 2]
```diff
@@ -76,10 +78,10 @@ bundle install
 
 *NOTE:* You may encounter issues installing gems. If you do, investigate the errors. You may require additional libraries that can be install with homebrew.
 
-Next, the database needs to be created. If Rails is installed correctly, you this command should create the `ceqr_rails` database and load the schema.
+Next, the database needs to be created. If Rails is installed correctly, you this command should create the `ceqr_rails` database, load the schema, and seed the db with some initial data.
```
Comment (godfreyyeung) [inline LOCALDEV.md]: nit: clarify `... correctly, you this command...`

Comment (bfreeds) [reply LOCALDEV.md]: Thanks for reviewing and finding these typos!  Fixed them both here:  https://github.com/NYCPlanning/ceqr-app/pull/501/commits/9016484811bddb3583d422eec08e2cb1f9e6448f.

**Comment by bfreeds** [comment]
@godfreyyeung thanks also for tracking down that issue you ran into running docker (and fixing it by commenting out `bootsnap/setup` in the ruby `boot.rb`.  I think that's a great idea to address that now so that hopefully it's a very smooth dev setup for @trbmcginnis.

I added another commit that by default would not use bootsnap (@pichot's comment indicates that it's just used to speed up the boot).

See here https://github.com/NYCPlanning/ceqr-app/pull/501/commits/7d047b21da0d822ffbdfe11a913ba9f1700cdc6b

**Pull request merged**